server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -80,16 +80,14 @@ export class ScribeLambda extends SequencedLambda {
         private protocolHead: number,
         messages: ISequencedDocumentMessage[],
     ) {
-        super(context);
-
         this.lastOffset = scribe.logOffset;
         this.setStateFromCheckpoint(scribe);
         // Filter and keep messages after protocol state
         this.pendingMessages = new Deque<ISequencedDocumentMessage>(
             messages.filter((message) => message.sequenceNumber > scribe.protocolState.sequenceNumber));
     }
 
-    public async handlerCore(message: IQueuedMessage): Promise<void> {
+    public async handler(message: IQueuedMessage) {
```

Comment:
Can we also change line 154 to be 
```
throw new Error(`Invalid message sequence number.`
                        + `Current message @${value.operation.sequenceNumber}.`
                        + `ProtocolHandler @${lastProtocolHandlerSequenceNumber}`);
```
instead of
```
this.context.error(new Error(`Invalid message sequence number`), {
                            restart: true,
                            tenantId: this.tenantId,
                            documentId: this.documentId,
                        });
```

That way I believe we can make this type of error surface to the `DocumentPartition` so we can mark the document as corrupt